### PR TITLE
fix(components/packages): add compat stylesheet to project source roots

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/726/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/726/storybooks/ag-grid-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/726/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/726/storybooks/layout-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/726/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/726/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/726/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/726/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/726/storybooks/tabs-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/726/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/726/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/726/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/726/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/726/playground/)

File: libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.spec.ts [hunk 1]
```diff
@@ -16,18 +16,18 @@ describe('Migrations > Add compat stylesheets', () => {
  * - Delete this file after all blocks have been addressed.
  * - Remove each occurrence of this file in your project's
  *   angular.json file.
-*******************************************************************************/
+ *******************************************************************************/
```
Comment (Blackbaud-PaulCrowder) [inline libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.spec.ts:19]: Was the addition of this space intentional?

File: libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.ts [hunk 2]
```diff
@@ -28,10 +29,19 @@ The preset bottom margin has been removed from alert components. To implement th
...
+  } else {
+    return `${projectDefinition.root}/src/app`;
+  }
+}
+
 function buildCommentBlock(message: string): string {
   return `/${'*'.repeat(79)}
  * ${message.replace(/(?![^\n]{1,75}$)([^\n]{1,75})\s/g, '$1\n * ')}
-${'*'.repeat(79)}/`;
+ ${'*'.repeat(79)}/`;
```
Comment (Blackbaud-PaulCrowder) [inline libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.ts:44]: Same question about the space here.

File: libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.ts [hunk 3]
```diff
@@ -93,30 +107,27 @@ function addStylesheetToWorkspace(): Rule {
...
-        packageJson.devDependencies?.[library.name]
-      ) {
-        for (const component of library.components) {
-          contents += buildComponentCss(component);
-        }
+function getCompatStyles(tree: Tree): string | undefined {
+  let contents = '';
+
+  const packageJson: {
+    dependencies?: { [_: string]: string };
```
Comment (Blackbaud-PaulCrowder) [inline libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.ts]: While you're in there, could you change these to `Record<string, string>`?

File: libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.spec.ts (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.spec.ts:19]: Yep. It's not super important, but it converts the comment block format from:
```
/**
 *
*******/
```

To:
```
/**
 *
 *******/
```
Which is more in line with how JSDocs creates their comment blocks. I can revert if you'd like.

File: libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.ts (hunk 3 above)
Comment (Blackbaud-SteveBrush) [reply libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.ts]: Good call. Done.

**Review by Blackbaud-PaulCrowder — APPROVED** [review]

**Pull request merged**